docs/content/integrations/embedded-elt.mdx

```diff
@@ -7,251 +7,11 @@ description: Lightweight ELT framework for building ELT pipelines with Dagster,
-  object: "{stream_schema}_{stream_table}"
-
-streams:
-  public.accounts:
-  public.users:
-  public.finance_departments:
-    object: "departments"
-```
-
-Or in Python:
-
-```python file=/integrations/embedded_elt/replication_config.py
-replication_config = {
-    "SOURCE": "MY_POSTGRES",
-    "TARGET": "MY_DUCKDB",
-    "defaults": {"mode": "full-refresh", "object": "{stream_schema}_{stream_table}"},
-    "streams": {
-        "public.accounts": None,
-        "public.users": None,
-        "public.finance_departments": {"object": "departments"},
-    },
-}
-```
-
----
-
-## Step 2: Create a Sling resource
-
-Next, you will need to create a <PyObject module="dagster_embedded_elt.sling" object="SlingResource" /> object that contains references to the connections specified in the replication configuration.
-
-A <PyObject module="dagster_embedded_elt.sling" object="SlingResource" /> takes a `connections` parameter, where each <PyObject module="dagster_embedded_elt.sling" object="SlingConnectionResource" /> represents a connection to a source or target database. You may provide as many connections to the `SlingResource` as needed.
-
-The `name` parameter in the <PyObject module="dagster_embedded_elt.sling" object="SlingConnectionResource" /> should match the `SOURCE` and `TARGET` keys in the replication configuration.
-
-You may pass a connection string or arbitrary keyword arguments to the <PyObject module="dagster_embedded_elt.sling" object="SlingConnectionResource" /> to specify the connection details. See the <a href="https://docs.slingdata.io/connections/database-connections">Sling connections reference</a> for the specific connection types and parameters.
-
-```python file=/integrations/embedded_elt/sling_connection_resources.py
-from dagster_embedded_elt.sling.resources import (
-    SlingConnectionResource,
-    SlingResource,
-)
-
-from dagster import EnvVar
-
-sling_resource = SlingResource(
-    connections=[
-        # Using a connection string from an environment variable
-        SlingConnectionResource(
-            name="MY_POSTGRES",
-            type="postgres",
-            connection_string=EnvVar("POSTGRES_CONNECTION_STRING"),
-        ),
-        # Using a hard-coded connection string
-        SlingConnectionResource(
-            name="MY_DUCKDB",
-            type="duckdb",
-            connection_string="duckdb:///var/tmp/duckdb.db",
-        ),
-        # Using a keyword-argument constructor
-        SlingConnectionResource(
-            name="MY_SNOWFLAKE",
-            type="snowflake",
-            host=EnvVar("SNOWFLAKE_HOST"),
-            user=EnvVar("SNOWFLAKE_USER"),
-            role="REPORTING",
-        ),
-    ]
-)
-```
-
----
-
-## Step 3: Define the Sling assets
+1. <a href="https://slingdata.io">Sling</a> to provide a simple way to sync data
+   between databases and file systems.
+2. <a href="https://dlthub.com">Data Load Tool (dlt)</a> to easily load data from
+   external systems and APIs.
```

Comment:
Should link to the two sub-pages here as well.

Reply:
Added links with short descriptions here: https://github.com/dagster-io/dagster/pull/20763/commits/24fd70ad1548d55893be2581be8fefc84660f24b